easybuild/easyconfigs/s/supermagic/supermagic-20220308-gompi-2021a.eb

```
@@ -0,0 +1,34 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'ConfigureMake'
+
+name = 'supermagic'
+version = '20220308'
+
+homepage = 'https://github.com/hpc/supermagic'
+description = "Very simple MPI sanity code. Nothing more, nothing less."
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+toolchainopts = {'usempi': True}
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('M4', '1.4.18'),
+]
+
+github_account = 'hpc'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': '481f3d9362.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['5777b613b1891ba76144d81139c57a7729eef38c4323b1882e132b2de5be919a']
+
+preconfigopts = "./autogen && "
+configopts = 'CC=$CC'
+
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': [],
+}
+
```

maybe add a simple sanity check command?
```suggestion

sanity_check_commands = ["OMPI_MCA_rmaps_base_oversubscribe=1 mpiexec -n 4 supermagic -t hello_world"]

```